src/components/TimeTable.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Table} from 'reactstrap'
+import ButtonTimeAvailable from 'components/ButtonTimeAvailable.jsx';
+import ButtonTimeDelete from 'components/ButtonTimeDelete.jsx';
+import 'components/TimeTable.css';
+
+export default class TimeTable extends React.Component{
+
+	render(){
+		return (
+     		<Table bordered>
+        		<thead>
+          			<tr>
```

Comment:
Render the html depend on state. Don't hard code!!
You could write other function which return the html, depending on the states.
and then call the function in "Main render" to complete the whole render.